From 2ee42fa713253059678ddf9f85ce5a4d4ee47272 Mon Sep 17 00:00:00 2001
From: Sunil Goutham <sgoutham@cavium.com>
Date: Thu, 24 Jul 2014 12:48:23 +0200
Subject: [PATCH 174/430] ITS: Removing debug prints

The commit comes from
  git://git.kernel.org/pub/scm/linux/kernel/git/rric/linux.git

Remove noisy debug printouts.

Signed-off-by: Sunil Goutham <sgoutham@cavium.com>
Signed-off-by: Robert Richter <rrichter@cavium.com>
(cherry picked from commit ac2e2998958fb86c50283c84e46f099a56afde70)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/irqchip/irq-gic-v3-its.c |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/drivers/irqchip/irq-gic-v3-its.c b/drivers/irqchip/irq-gic-v3-its.c
index 1549587..6ad85d7 100644
--- a/drivers/irqchip/irq-gic-v3-its.c
+++ b/drivers/irqchip/irq-gic-v3-its.c
@@ -1202,7 +1202,6 @@ static int its_msi_setup_irq(struct msi_chip *chip,
 		if (WARN_ON(nvec <= 0))
 			return nvec;
 		its_dev = its_create_device(its, dev_id, nvec);
-		dev_info(&pdev->dev, "ITT %d entries, %d bits\n", nvec, ilog2(nvec));
 	}
 	if (!its_dev)
 		return -ENOMEM;
@@ -1211,7 +1210,6 @@ static int its_msi_setup_irq(struct msi_chip *chip,
 	if (err)
 		return err;
 
-	dev_info(&pdev->dev, "ID:%d pID:%d vID:%d\n", its_msi_get_entry_nr(desc), hwirq, irq);
 	irq_set_msi_desc(irq, desc);
 	irq_set_handler_data(irq, its_dev);
 
-- 
1.7.5.4

